dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -936,6 +1140,93 @@ def test_json_decode_after_update(self):
 
         test_utils.assert_profiles_equal(deserialized, expected_profile)
 
+    def test_cms_heavy_hitters_threshold(self):
+
+        df_categorical = pd.Series(
+            [
+                "a",
+                "a",
+                "a",
+                "a",
+                "a",
+                "b",
+                "b",
+                "b",
+                "b",
+                "b",
+                "c",
+                "c",
+                "c",
+                "c",
+                "c",
+                "c",
+                "c",
+                "c",
+                "c",
+                "c",
```

Comment:
I like in the below where you use `["a"] * ...+ ...`, makes it clear how many to expect for each.

Reply:
looks like 5,5, 10 respectively